tests/real_git.rs

```diff
@@ -75,3 +79,21 @@ fn can_list_all_branches() {
     let branches = git.all_branches().unwrap();
     assert!(branches.contains("trunk"));
 }
+
+// Cleaning PRs requires that we identify "old" branches (those which have been merged into trunk),
+// and that we delete those branches. Because the tests run in parallel, we need to ensure that our
+// check for the existence of the "hotfix" branch always happens *before* our attempt to delete the
+// "hotfix" branch. So this test case exercises all the git client functionality we would need in
+// order to implement the "pr-clean" subcommand.
+#[test]
+fn could_clean() {
+    println!("TempDir='{:?}'", TEST_STATE.path());
```

Comment:
Would be nice to push this into a macro I think to hide "touching" of tempdir. Maybe "init_test_state"? Or something like that.